internal/cmds/cmds.go

```diff
@@ -34,6 +34,8 @@ var (
 		cs: newCommandSlice([]string{"ROLE"}),
 	}
 	// QuitCmd is predefined QUIT
+	// We shouldn't use this because its deprecated
+	// https://github.com/redis/rueidis/issues/377
 	QuitCmd = Completed{
```

Comment:
Please just remove the `QuitCmd`

Reply:
Should I also remove this QUIT in builder ?
https://github.com/redis/rueidis/blob/6a804776c5e2b6e9582d3873e8806342b1751f75/internal/cmds/gen_connection.go#L1198-L1207

Reply:
No. We should keep the builder one.